### ISSUE TITLE
Column headers `p_linear_trend` and `linear_trend` in summary table (`write_summary_table`)

### ISSUE THREAD

**Issue by DagHjermann** [osparcomm/HARSAT#483]
The column headers `p_linear_trend` and `linear_trend` seems misleading to me. It seems to me that these are the statistics for the whole time series, `pltrend` (as opposed to `prtrend` for the 'recent' time series = last 20 years)? 

If so, I think these column headers (marked in red below) could be renamed to for instance `p_overall_trend` and `overall_trend` (or `p_whole_trend` and `whole_trend`). [See code](https://github.com/osparcomm/HARSAT/blob/744a2afb1f4f684566854590d7d71589188e38a9/R/reporting_functions.R#L666-L667).

See example below, which are results from a "U-shaped" trend, so is no change from the start to the stop of the time series, but there is a significant increase for the last 20 years. Top: the header from the summary file. Bottom: tables and text from the HTML file. 
![image](https://github.com/user-attachments/assets/562d28c0-5710-4443-a88b-b4df8a935e4f)
Output from HTML file, "statistics" tab:
![image](https://github.com/user-attachments/assets/763f9072-8438-4446-95b4-e3af51bfecdc)
![image](https://github.com/user-attachments/assets/03ddffc8-7867-4b73-9829-40104949bd99)

**Comment by RobFryer** [comment]
@DagHjermann 
Yes, completely agree.  I think I will use `p_overall_trend` and `overall_trend` rather than `p_whole_trend` and `whole_trend` because that ties in better with the contrast table (change in log concentration).  
If it's ok with you, I'll make the change after the next release.  The summary tables are used elsewhere in the OSPAR 2024 assessment (material for the OHAT and for the DOIs - which is not in harsat yet) and I don't want to have to check whether the change in `write_summary_table` has knock on effects.  
I've also got a full description of all the output variables which should go into the help file, but again I'll leave that for now - running out of time!

**Comment by RobFryer** [comment]
@DagHjermann 
`p_overall_change` and `p_recent_change` would be more accurate perhaps?  (Still expressed as percentage change per year.)  But maybe less intuitive.  What do you think?

**Comment by DagHjermann** [comment]
`p_overall_change` is certainly more accurate than `_trend`, I think! Perhaps slightly less intuitive but I think this is the better option, as it better reflects what is done, mathematically. The version `p_overall_trend` can also be confusing ("there is a significant non-linear trend" but still "`p_overall_trend` > 0.05"?).
Completely ok with me (us) that the change is done after the next release!

**Comment by annelaerkes** [comment]
I think this is a good discussion and it would be great if the headings were more intuitative than they are now. I am currently struggling with understanding what is in each of the output variables in the output table (including the "power" columns). Is there anywhere this is better described than in the AMAP Hg report pag 61 (which was what I got from Simon). I am extracting statistics for our yearly national report but unsure what to include in our tables.
